model/installation_request.go

```diff
@@ -369,6 +380,20 @@ func (p *PatchInstallationRequest) Apply(installation *Installation) bool {
 		applied = true
 		installation.License = *p.License
 	}
+
+	if p.OverrideIPRanges != nil && *p.OverrideIPRanges != installation.OverrideIPRanges {
+		applied = true
+		installation.OverrideIPRanges = *p.OverrideIPRanges
```

Comment:
I am wondering if we need to store the override bool on the installation itself. It makes sense that we need to be able to pass in an override bool on the patch, but then it seems like patch would just force the new IP range value on the installation and then it could be disregarded.

Example:
1. Set allowed IP range to `1.1.1.1/24`, store on installation and apply to ingress
2. Sometime later, patch installation to IP range `2.2.2.2/24` and use force override. At that time, store the new installation range and apply to ingress.

I see that we use the override in a few other places, but unless I am missing something this would be a simpler architecture. There are similar patching mechanisms like the `ClearOrPatch` logic below.